app/models/organisation.rb

```diff
@@ -2,6 +2,6 @@ class Organisation < ApplicationRecord
   has_many :users
   has_many :funds
 
-  validates_presence_of :name, :organisation_type, :language_code, :default_currency
+  validates_presence_of :name, :organisation_type, :language_code, :default_currency, :iati_reference
```

Comment:
I did consider it, and landed on leaving it out, mainly through my lack of knowledge about the code itself and how difficult it will be for users to know what it is and/or get one that is actually unique - I think your reasoning is strong though so let's pop it in!